lib/srv/exec_linux_test.go

```diff
@@ -132,6 +137,36 @@ func TestConfigureCommand(t *testing.T) {
 	require.NotContains(t, cmd.Env, unexpectedKey+"="+unexpectedValue)
 }
 
+func TestRootConfigureCommand(t *testing.T) {
+	utils.RequireRoot(t)
+
+	login := "teleport-test-user"
+	_, err := host.UserAdd(login, nil, "", "", "")
```

Comment:
Will this fail if the user exists?

Could we avoid creating a new OS user just for testing?

Reply:
I'm not sure how to avoid creating a user for the test given that the behavior between multiple users is exactly what we're testing, but I've updated this to generate a new login each time so pre-existing users won't be a problem.

Reply:
If we could cleanup after ourselves that would be best.

Reply:
We do a few lines down. https://github.com/gravitational/teleport/blob/5ca7c657005ef3201562adfdbbf4c92c0ee23fc1/lib/srv/exec_linux_test.go#L146-L149